### PR TITLE
feature: implement PullImage and ListImages methods for cri manager

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

**1.Describe what this PR did**

**2.Does this pull request fix one issue?** 

fixes part of #420

**3.Describe how you did it**

**4.Describe how to verify it**

Maybe the [cri-tools](https://github.com/kubernetes-incubator/cri-tools) is what you need.

We may use the following method to verify that we can pull and list images:
```
[root@pouch-test pouch]# crictl pull docker.io/library/redis:alpine
[root@pouch-test pouch]# crictl images
IMAGE                       TAG                 IMAGE ID            SIZE
docker.io/library/redis     alpine              0153c5db97e58       10.1MB
```

**5.Special notes for reviews**

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@skoo87 @allencloud PTAL.

File: daemon/mgr/cri.go [hunk 1]
```diff
@@ -247,9 +249,41 @@ func (c *CriManager) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 	return nil, fmt.Errorf("Status Not Implemented Yet")
 }
 
+func imageToRuntimeAPIImage(image *apitypes.ImageInfo) (*runtime.Image, error) {
```
Comment (allencloud) [inline daemon/mgr/cri.go]: Could you add a comment for this function?
Actually I have no strong idea about RuntimeAPIImage.
Thanks. 😄

**Comment by YaoZengzeng** [comment]
@allencloud  updated.

File: daemon/mgr/cri.go [hunk 2]
```diff
@@ -259,7 +294,24 @@ func (c *CriManager) ImageStatus(ctx context.Context, r *runtime.ImageStatusRequ
 
 // PullImage pulls an image with authentication config.
 func (c *CriManager) PullImage(ctx context.Context, r *runtime.PullImageRequest) (*runtime.PullImageResponse, error) {
-	return nil, fmt.Errorf("PullImage Not Implemented Yet")
+	// TODO: authentication.
+	imageRef := r.GetImage().GetImage()
```
Comment (allencloud) [inline daemon/mgr/cri.go]: It makes feel weird, although it is correct totally. 😄 
double GetImage().

**Comment by allencloud** [comment]
LGTM, I think we could begin considering adding API integration test. @YaoZengzeng @Letty5411

**Pull request merged**